CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 The release is a more complete alpha version meant to be deployed on the second iteration of the public testnet.
 There are no guarantees on future API and binary compatibility at this stage.
 
+## v0.1.1
+
+### Features
+* *client*: export and import transaction -- transaction in which the receiver's view key not included can be exported by a give transaction id (the sender can list transactions and find it), getting a base64 encoded plain transaction string and the receiver can import the string into wallet.
```

Comment:
master is already unreleased 0.2, so perhaps no "0.1.1", just move the features paragraph under 0.2